lib/lumberjack/server.rb

```diff
@@ -138,10 +141,10 @@ def header(&block)
       version, frame_type = get.bytes.to_a[0..1]
 
       case frame_type
-        when FRAME_WINDOW; transition(:window_size, 4)
-        when FRAME_DATA; transition(:data_lead, 8)
-        when FRAME_COMPRESSED; transition(:compressed_lead, 4)
-        else; raise "Unknown frame type: #{frame_type}"
+      when FRAME_WINDOW; transition(:window_size, 4)
```

Comment:
This ... non-indented 'case/when' style is hard to read, but we'll discuss that in the future when we get together and figure out rubocop styles.